libyul/optimiser/Suite.cpp

```diff
@@ -383,6 +383,24 @@ void OptimiserSuite::validateSequence(std::string_view _stepAbbreviations)
 	assertThrow(nestingLevel == 0, OptimizerException, "Unbalanced brackets");
 }
 
+bool OptimiserSuite::isEmptyOptimizerSequence(std::string const& _sequence)
+{
+	size_t delimiterCount{0};
+	for (char const step: _sequence)
+		switch (step)
+		{
+		case ':':
+			++delimiterCount;
+			break;
```

Comment:
We can stop iterating as soon as we see the second delimiter.
```suggestion
			if (delimiterCount >= 2)
				return false;
```
Then `return true` at the end of the function.